docs/sources/setup/install/helm/install-scalable/_index.md

```diff
@@ -269,34 +236,21 @@ write:
 minio:
   enabled: false
 
-# Zero out replica counts of other deployment modes
-singleBinary:
-  replicas: 0
-
-ingester:
-  replicas: 0
-querier:
-  replicas: 0
-queryFrontend:
-  replicas: 0
-queryScheduler:
-  replicas: 0
-distributor:
-  replicas: 0
-compactor:
-  replicas: 0
-indexGateway:
-  replicas: 0
-bloomCompactor:
-  replicas: 0
-bloomGateway:
-  replicas: 0
-```
-
 {{< /code >}}
 
 To configure other storage providers, refer to the [Helm Chart Reference]({{< relref "../reference" >}}).
 
+## Deploying the Loki Helm chart to a Production Environment
+
+{{< admonition type="note" >}}
+We are activley working on providing more guides for deploying Loki in production. 
+{{< /admonition >}}
+
+It is recommended to run Loki at scale within in a cloud enviroment like AWS, Azure, or GCP. The below guides will show you how to deploy a minimally viable production environment.
```

Comment:
Passive language.  "We recommend"

https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#write-in-active-voice